### PR TITLE
Implement SetTenantLabel mutation

### PR DESCRIPTION
**Pull request by kaloyanspiridonov**
**Description**

Changes proposed in this pull request:
- Introduce a new mutation, SetTenantLabel, that allows setting labels on tenants.
- E2E test that validates that the mutation is working as expected

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/pkg/graphql/schema.graphql
Comment (dragobt) [inline components/director/pkg/graphql/schema.graphql:1]: update dependencies as there are changes in the pkg folder and bump values versions

**Review by dragobt — CHANGES_REQUESTED** [review]

**Review by dragobt — APPROVED** [review]

**Review by StanislavStefanov — APPROVED** [review]

**Pull request merged**